### PR TITLE
Complete Space Travelers Hub App

### PR DESCRIPTION
**Pull request by see-why**
- Install [React Redux](https://react-redux.js.org/), [Redux Logger](https://www.npmjs.com/package/redux-logger) and [React Router](https://reactrouter.com/web/guides/quick-start).
- Download the [free image](https://www.flaticon.com/free-icon/planet_3212567?term=space&page=1&position=19&page=1&position=19&related_id=3212567&origin=style) for the app logo.
- Create routes and view components (Rockets and Missions). 
- Use `<NavLink />` for the page navigation links and style active class to indicate which section/page user is currently on (underline active navigation link).
- Create directories for all Redux state slice files (rockets and missions).
- Upon first render fetch data from the SpaceX API endpoints:
    - Rockets: `https://api.spacexdata.com/v3/rockets`
    - Missions: `https://api.spacexdata.com/v3/missions`
- Once the data are fetched, dispatch an action to store the selected data in Redux store:
    - Rockets: 
        - id
        - rocket_name
        - description
        - flickr_images
    - Missions:
        - mission_id
        - mission_name
        - description 
- Use `useSelector()` Redux Hook to select the state slices and render lists of rockets and missions in corresponding routes.
- Render a list of rockets, use the first image in the array of `flickr_images`.
- Render a table with the missions' data.
- User clicks the "Reserve rocket" button, action needs to be dispatched to update the store. 
- Dispatch the action with the correct rocket ID as an argument.
- Create a reducer and action dispatcher for the "Join Mission" button, with the selected mission having a new key/value - `reserved: true`.
- Rockets that have already been reserved should show a "Reserved" badge and "Cancel reservation" button instead of the default "Reserve rocket".
- Missions that the user has joined already should show a badge "Active Member" instead of the default "NOT A MEMBER" and a button "Leave Mission" instead of the "Join Mission" button.
- Rockets and Missions should use the React conditional rendering syntax
- Compose two/three column layout and list ONLY the rockets reserved and missions joined by the user (as per design):
    - Render a list of all joined missions (use `filter()`).
    - Render a list of all reserved rockets (use `filter()`).
- Set up the repository on GitHub and use Gitflow.
- Set up React and Redux.
- Set up testing libraries.
- Do not preserve user data.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello Team,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+## Built With
+
+- React
+- JSX
+- Redux
+
+## Live Demo
+
+- [Live Demo Link]( https://rocket-react-hub.netlify.app/)
```
Comment (mikenath223) [inline README.md:14]: - [x] As stated in the project requirements, kindly record a video discussing the aspects listed in the requirements.

Comment (see-why) [reply README.md:14]: Hi @mikenath223 

Thanks for your speedy response.
However, I do not think the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md) stated we add a video.
Kindly review the requirements and guide us further.

**Comment by bellom** [comment]
Hello 👋,

This is an additional review as a result of the second opinion request.

You're right 😄 , sorry for the mixed up from the Code Reviewer 🙏 .

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away :)

Keep rocking 🚀!

**Comment by see-why** [comment]
@bellom 

Thanks a lot.

I will do as you have advised.
🙇‍♂️

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @see-why 🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by see-why** [comment]
@bimbolabuari 

Thanks a lot.
Do have a great day. 😃

**Pull request merged**